stdlib/pervasives.ml

```diff
@@ -521,7 +525,7 @@ external format_of_string :
  ('a, 'b, 'c, 'd, 'e, 'f) format6 ->
  ('a, 'b, 'c, 'd, 'e, 'f) format6 = "%identity"
 
-let ( ^^ ) (Format (fmt1, str1)) (Format (fmt2, str2)) =
+let (^^) (Format (fmt1, str1)) (Format (fmt2, str2)) =
```

Comment:
You should fix `threads/pervasives` to add the spaces rather than remove them here.